vita3k/gui/src/app_context_menu.cpp

```diff
@@ -468,11 +468,11 @@ void draw_app_context_menu(GuiState &gui, EmuEnvState &emuenv, const std::string
             if (!emuenv.cfg.show_live_area_screen && ImGui::BeginMenu("Live Area")) {
                 if (ImGui::MenuItem("Live Area", nullptr, &gui.vita_area.live_area_screen))
                     open_live_area(gui, emuenv, app_path);
-                if (ImGui::MenuItem("Search", nullptr))
+                if (ImGui::MenuItem(lang.main["search"].c_str(), nullptr))
                     open_search(APP_INDEX->title);
-                if (fs::exists(MANUAL_PATH) && !fs::is_empty(MANUAL_PATH) && ImGui::MenuItem("Manual", nullptr))
+                if (fs::exists(MANUAL_PATH) && !fs::is_empty(MANUAL_PATH) && ImGui::MenuItem(lang.main["manual"].c_str(), nullptr))
                     open_manual(gui, emuenv, app_path);
-                if (ImGui::MenuItem("Update"))
+                if (ImGui::MenuItem(lang.main["update"].c_str()))
```

Comment:
Sorry, for this is like i have explain you before, keep english word
(Size of button is fix) so no can get word more lenght
Is for that i have no translate it

Reply:
but it is in app_context.cpp, no add in live_area.cpp, i knew here is icon

Reply:
Oh 🗿 sorry forget lol